Makefile

```diff
@@ -17,15 +18,28 @@ build: generate
 test:
 	go test $(TEST_SOURCES_NO_CUCUMBER)
 
+UNITS = "@unit.sourcemap,@unit.offline,@unit.algod,@unit.indexer,@unit.transactions.keyreg,@unit.rekey,@unit.tealsign,@unit.dryrun,@unit.responses,@unit.applications,@unit.transactions,@unit.indexer.rekey,@unit.responses.messagepack,@unit.responses.231,@unit.responses.messagepack.231,@unit.responses.genesis,@unit.feetest,@unit.indexer.logs,@unit.abijson,@unit.abijson.byname,@unit.transactions.payment,@unit.atomic_transaction_composer,@unit.responses.unlimited_assets,@unit.indexer.ledger_refactoring,@unit.algod.ledger_refactoring,@unit.dryrun.trace.application"
 unit:
 	go test $(TEST_SOURCES_NO_CUCUMBER)
-	cd test && go test -timeout 0s --godog.strict=true --godog.format=pretty --godog.tags="@unit.sourcemap,@unit.offline,@unit.algod,@unit.indexer,@unit.transactions.keyreg,@unit.rekey,@unit.tealsign,@unit.dryrun,@unit.responses,@unit.applications,@unit.transactions,@unit.indexer.rekey,@unit.responses.messagepack,@unit.responses.231,@unit.responses.messagepack.231,@unit.responses.genesis,@unit.feetest,@unit.indexer.logs,@unit.abijson,@unit.abijson.byname,@unit.transactions.payment,@unit.atomic_transaction_composer,@unit.responses.unlimited_assets,@unit.indexer.ledger_refactoring,@unit.algod.ledger_refactoring,@unit.dryrun.trace.application" --test.v .
+	cd test && go test -timeout 0s --godog.strict=true --godog.format=pretty --godog.tags=$(UNITS) --test.v .
 
+INTEGRATIONS = "@algod,@assets,@auction,@kmd,@send,@indexer,@rekey_v1,@send.keyregtxn,@dryrun,@compile,@applications.verified,@indexer.applications,@indexer.231,@abi,@c2c,@compile.sourcemap"
```

Comment:
@tzaffi The annotation list here does not match https://github.com/algorand/py-algorand-sdk/pull/367/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R5.

Can you help deduce which (if either) listing is accurate?

Aside and open to discussion:  
* I've long resisted it because it's a pre-existing concern, but it may be time to rework the annotation listing to have 1 value per line alphabetized.
* It's extremely difficult to compare changes.  I can also see moving these annotations to a separate file.

Reply:
It _seems_ the only diff is py-algorand-sdk removes `@indexer.231` while this PR retains it.

Reply:
I'm attempting a new approach. LMK.

Reply:
@tzaffi Thanks - resolving!  https://github.com/algorand/go-algorand-sdk/pull/360/commits/9d842a99bd3bc9f646b4e4616673ebe1c1bdd8c5 is what I had in mind.  ☕